tests/integration/cqlengine/model/test_udts.py

```diff
@@ -0,0 +1,248 @@
+# Copyright 2015 DataStax, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import date, datetime
+from uuid import UUID
+from decimal import Decimal
+import pytz
+
+from cassandra.cqlengine.models import Model
+from cassandra.cqlengine.usertype import UserType
+from cassandra.cqlengine import columns
+from cassandra.cqlengine.management import sync_table, sync_type, create_keyspace_simple, drop_keyspace
+
+from tests.integration.cqlengine.base import BaseCassEngTestCase
+
+class UserDefinedTypeTests(BaseCassEngTestCase):
+
+    def test_can_create_udts(self):
+        class User(UserType):
+            age = columns.Integer()
+            name = columns.Text()
+
+        sync_type("cqlengine_test", User)
+        user = User(age=42, name="John")
+        self.assertEqual(42, user.age)
+        self.assertEqual("John", user.name)
+
+        # Add a field
+        class User(UserType):
+            age = columns.Integer()
+            name = columns.Text()
+            gender = columns.Text()
+
+        sync_type("cqlengine_test", User)
+        user = User(age=42, name="John", gender="male")
+        self.assertEqual(42, user.age)
+        self.assertEqual("John", user.name)
+        self.assertEqual("male", user.gender)
+
+        # Remove a field
+        class User(UserType):
+            age = columns.Integer()
+            name = columns.Text()
+
+        sync_type("cqlengine_test", User)
+        user = User(age=42, name="John", gender="male")
+        with self.assertRaises(AttributeError):
+            user.gender
+
+    def test_can_insert_udts(self):
+        class User(UserType):
+            age = columns.Integer()
+            name = columns.Text()
+
+        class UserModel(Model):
+            id = columns.Integer(primary_key=True)
+            info = columns.UserDefinedType(User)
+
+        sync_table(UserModel)
+
+        user = User(age=42, name="John")
+        UserModel.create(id=0, info=user)
+
+        self.assertEqual(1, UserModel.objects.count())
+
+        john = UserModel.objects().first()
+        self.assertEqual(0, john.id)
+        self.assertTrue(type(john.info) is User)
```

Comment:
This doesn't work on my stack, even though ``type(john.info)`` is ``<class 'tests.integration.cqlengine.model.test_udts.User'>``.
Does this assertion work for you?